compiler/ballerina-lang/src/main/java/org/ballerinalang/util/EmbeddedExecutorProvider.java

```diff
@@ -10,6 +10,11 @@
  */
 public class EmbeddedExecutorProvider {
     private static EmbeddedExecutorProvider provider;
+
+    static {
+        provider = new EmbeddedExecutorProvider();
```

Comment:
why static block, cant we do a single line in line 12.

Reply:
Fixed with 9fcdece5175329f9bb65d427cb38db2c9f44d701